public/pages/ManagedIndices/containers/ManagedIndices/ManagedIndices.tsx

```diff
@@ -561,18 +541,24 @@ export class ManagedIndices extends MDSEnabledComponent<ManagedIndicesProps, Man
     );
 
     const popoverActionItems = [
-      <EuiContextMenuItem
-        key="Edit"
-        toolTipPosition="left"
-        disabled={selectedItems.length !== 1 || isDataStreamIndexSelected}
-        data-test-subj="editOption"
-        onClick={() => {
-          this.closePopover();
-          this.onShowEditModal();
-        }}
-      >
-        Edit rollover alias
-      </EuiContextMenuItem>,
+      <ModalConsumer>
+        {({ onShow, onClose }) => (
+          <EuiContextMenuItem
+            key="Edit"
+            toolTipPosition="left"
+            disabled={selectedItems.length !== 1 || isDataStreamIndexSelected}
+            data-test-subj="editOption"
+            onClick={() =>
+              onShow(RolloverAliasModal, {
+                index: selectedItems[0].index,
+                core: this.context,
+              })
+            }
+          >
+            Edit rollover alias
+          </EuiContextMenuItem>
+        )}
+      </ModalConsumer>,
```

Comment:
Any reason we are changing the logic here?

Reply:
There was a bug in that implementation there due to which the button was not working properly, so have fixed that